examples/bin/cluster-syntheticcontrol.sh

```diff
@@ -75,11 +75,21 @@ if [ "$HADOOP_HOME" != "" ] && [ "$MAHOUT_LOCAL" == "" ]; then
     echo "DFS is healthy... "
     echo "Uploading Synthetic control data to HDFS"
     $DFSRM ${WORK_DIR}/testdata
-    $DFS -mkdir ${WORK_DIR}/testdata
+    $DFS -mkdir -p ${WORK_DIR}/testdata
```

Comment:
the problem with `$DFS -mkdir -p` is that it's not backwards compatible with Hadoop 1, which Mahout still technically supports.